recipes/rnaz/build.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-./configure CFLAGS="-std=gnu89 -g -O2" --prefix=$PREFIX --datadir=$PREFIX/share
-make
+./autogen.sh
+./configure --prefix=$PREFIX --datadir=$PREFIX/share
```

Comment:
Did you test if removing the -O2 flag changes the executable speed?

Reply:
I've fixed that and used $CFLAGS, which uses -O2

Reply:
guys, why is this flag back, as far as I see I removed it with my last commit?

Reply:
-O2 is effectively the only one added back, to enable a bit of optimization.

Reply:
fine for me, was just wondering why it popped up in the review, thanks for checking